libraries/Microsoft.Bot.Builder.Dialogs.Rules/Steps/HttpRequest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Rules.Steps
+{
+    public class HttpRequest : DialogCommand
+    {
+
+        public enum Method {
+            GET,
+            POST
+        }
+
+        public HttpRequest()
+            : base()
+        {
+        }
+
+        protected override string OnComputeId()
+        {
+            return $"HttpRequest[{method}]";
+        }
+
+        public Method method;
+
+        public string url;
```

Comment:
This is a general rule for all this codebase. If this is public, please make it a property (i.e. add {get';set;}) and make the name Method. If it is private, make it private, but no public member variables.